include/mbedtls/pk.h

```diff
@@ -696,12 +634,23 @@ int mbedtls_pk_sign(mbedtls_pk_context *ctx, mbedtls_md_type_t md_alg,
                     unsigned char *sig, size_t sig_size, size_t *sig_len);
 
 /**
- * \brief           Make signature given a signature type.
+ * \brief           Restartable version of \c mbedtls_pk_sign()
+ *
+ * \note            Performs the same job as \c mbedtls_pk_sign(), but can
+ *                  return early and restart according to the limit set with \c
+ *                  psa_interruptible_set_max_ops() to reduce blocking for ECC
+ *                  operations. For RSA, same as \c mbedtls_pk_sign().
+ *
+ * \note            See the note on \c mbedtls_pk_sign() for which algorithm is
+ *                  used. If you want to select a specific signature algorithm
+ *                  since the is no restartable equivalent of \c
+ *                  mbedtls_pk_sign_ext(), you'll want to import the key into
+ *                  PSA using \c mbedtls_pk_import_into_psa() and then call PSA
+ *                  interruptible functions directly.
```

Comment:
Given that wrapped PSA keys aren't supported (which should be mentioned in this function's description, btw), this simplifies to “this function always uses `MBEDTLS_PK_ALG_ECDSA`” (plus being equivalent to `mbedtls_pk_sign` for RSA keys as stated just above).